management/schemas/distance-target.yaml

```diff
@@ -0,0 +1,74 @@
+schemas:
+
+  DistanceTarget:
+    type: object
+    properties:
+      Id:
+        type: integer
+        format: int32
+      FlightId:
+        type: integer
+        format: int32
+      Distance:
+        type: number
+        format: float
+        nullable: true
+      Latitude:
+        type: number
+        format: float
+        nullable: true
+      Longitude:
+        type: number
+        format: float
+        nullable: true
+      StreetAddress:
+        type: string
+        nullable: true
+
+  BatchUpload:
```

Comment:
For future reference, the names of these things wind up turning into class names if people generate code from the spec.

`BatchUpload` isn't a very descriptive name.

Reply:
What about `BatchUploadDistanceTargeting` ?

Reply:
Updated